src/tox.rs

```diff
@@ -24,6 +24,61 @@ pub enum Answer {
     Event(Event),
 }
 
+fn get_peer_info(
+    tox: &mut rstox::core::Tox,
+    conference: u32,
+    peer: u32
+) -> Result<PeerInfo, ConferencePeerQueryError> {
+    let pk = tox.get_peer_public_key(conference, peer)
+        .map_err(|e| e.try_into().unwrap())?;
```

Comment:
try_into?

Reply:
Yeah, all error conversions use TryInto. This is done because some errors do not exist in ws tox protocol (for example, `NullError`).

Reply:
Well.. better implement convertion from NullError than .unwrap and fail?

Reply:
`ConferencePeerQueryError` is fully covered, so this can not fail. Also, since you cannot send a null pointer over the internet, you are pretty much protected from a `NullError`.